server/routes/project.routes.js

```diff
@@ -26,4 +26,6 @@ router.get('/:username/projects', ProjectController.getProjectsForUser);
 
 router.get('/projects/:project_id/zip', ProjectController.downloadProjectAsZip);
 
+router.patch('/project/visibility', ProjectController.changeProjectVisibility);
```

Comment:
Not sure if we need a separate endpoint?  Can we use the "update" endpoint?

Reply:
I guess it will be better to have its separate endpoint. I'm not sure where we want to put that code without making another function messy. There's already a lot going on inside the `updateProject` function.